internal/server/server.go

```diff
@@ -116,7 +116,8 @@ func New(logger Logger, client Client, templates template.Templates, prefix, sir
 	mux.Handle("/delete-fee-reduction", wrap(DeletePayment(client, templates.Get("delete_fee_reduction.gohtml"))))
 	mux.Handle("/edit-payment", wrap(EditPayment(client, templates.Get("edit_payment.gohtml"))))
 	mux.Handle("/edit-fee-reduction", wrap(EditFeeReduction(client, templates.Get("edit_fee_reduction.gohtml"))))
-	mux.Handle("/payments", wrap(GetPayments(client, templates.Get("payments.gohtml"))))
+	mux.Handle("/payments/{id}", wrap(GetPayments(client, templates.Get("payments.gohtml"))))
+	mux.Handle("/mlpa/{uid}/payments", wrap(GetPayments(client, templates.Get("mlpa-payments.gohtml"))))
```

Comment:
Can we make this `/lpa/` to match the `/lpa/{uid}` endpoint?
```suggestion
	mux.Handle("/lpa/{uid}/payments", wrap(GetPayments(client, templates.Get("mlpa-payments.gohtml"))))
```